src/common/index.js

```diff
@@ -1,11 +1,25 @@
+import {Button} from './Button';
+import DashboardSkeleton from './DashboardSkeleton';
+import FilterComponent from './FilterComponent';
 import Footer from './Footer';
+import Image from './Image';
+import ImageGrid from './ImageGrid';
 import Navbar from './Navbar';
-import {Button} from './Button';
+import PopUpComponent from './PopUpComponent';
+import StatsComponent from './StatsComponent';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/n4alphapesto/hirebook-frontend/pullRequest?prid=7756469)
```suggestion
import StatsComponent from "./StatsComponent";
```